tests/sentry/api/endpoints/test_relay_projectconfigs.py

```diff
@@ -45,13 +45,13 @@ def private_key(key_pair):
 
 @pytest.fixture
 def relay_id():
-    return six.binary_type(six.text_type(uuid4()).encode("ascii"))
+    return six.text_type(uuid4())
```

Comment:
Having this as binary seems to produce authentication issues in the endpoint. Likely because what comes back from the model that's inserted is a string, not a bytestring. So it will fail to compare them.

Reply:
Hm, though actually this might need to be decoded in the actual request check. 🤔